htsget-http-actix/src/main.rs

```diff
@@ -14,6 +15,9 @@ This executable doesn't use command line arguments, but there are some environme
 * HTSGET_IP: The ip to use. Default: 127.0.0.1
 * HTSGET_PORT: The port to use. Default: 8080
 * HTSGET_PATH: The path to the directory where the server should be started. Default: Actual directory
+* HTSGET_REGEX: The regular expression that should match an ID. Default: ".*"
```

Comment:
Aha, I see now, `HTSGET_MATCH` & `HTSGET_SUBSTITUTION`... perhaps even with `_REGEX_` in the middle to make it more explicit? It took me a bit longer to grasp what it meant otherwise.

Reply:
Yeah, it is probably better that way 👍🏻